examples/node/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "test": "mocha -R spec --recursive test/helper.js test/lib/",
-    "test:debug": "npm test -- --debug-brk"
+    "test:debug": "yarn test -- --debug-brk"
```

Comment:
Not suggesting but mentally laughing at the thought of making every npm script agnostic to npm vs yarn.

```
"config": {"pkger": "yarn"},
"scripts": {"test:debug": "$npm_package_config_pkger test -- --debug-brk"}
```